.github/workflows/developer.yml

```diff
@@ -54,3 +54,9 @@ jobs:
         path: |
           bufr/build/*.html
           bufr/build/*.css
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: docs
+        path: |
+          bufr/build/docs/html
```

Comment:
Neat! I will add this elsewhere...

Reply:
Yes, really handy! Also note that doxygen is now adding information to the top of the Doxyfile showing what version of doxygen was last used to update it, which is helpful. So right now it says:

`# Doxyfile 1.9.1
`
on the first line of the file.

In my experience it is not necessary to update for every doxygen release. It only needs to be updated if there is a newer feature we want to use (as with WARN_AS_ERROR), or if it starts generating warnings about an out of date Doxyfile.

But it is harmless to update any time you feel like it as well. So feel free to run doxygen -u Doxyfile.in with new versions of doxygen as ubuntu-latest is changed.

Also in many of my developer CI builds I have the command doxygen --version before the build happens. That way if there is a problem I can instantly check what version of doxygen was used by the CI system.